corehq/apps/userreports/rebuild.py

```diff
@@ -0,0 +1,46 @@
+from corehq.apps.userreports.models import id_is_static
+from dimagi.utils.couch import get_redis_client
+
+
+def get_redis_key_for_config(config):
+    if id_is_static(config._id):
+        rev = 'static'
+    else:
+        rev = config._rev
+    return 'ucr_queue-{}:{}'.format(config._id, rev)
+
+
+class DataSourceResumeHelper(object):
+
+    def __init__(self, config):
+        self.config = config
+        self._client = get_redis_client().client.get_client()
+        self._key = get_redis_key_for_config(config)
+
+    def get_ids_to_resume_from(self):
+        # maintaining support for existing sets in redis while the
+        # transition to lists occur
+        # todo: is this still necessary?
```

Comment:
I doubt that's necessary I think @calellowitz added this a few months ago? shouldn't have any old ones laying around